blockchain/network.py

```diff
@@ -545,10 +523,10 @@ def phase_3_message(self, phase_3):
         phase_3_info = {
             RECORD: thrift_record_to_dict(phase_3.record),
             VERIFICATION_INFO: {
-                'lower_phase_hashes': phase_3.lower_phase_hashes,
-                'p2_count': phase_3.p2_count,
-                'business_list': phase_3.business_list,
-                'deploy_location_list': phase_3.deploy_loc_list
+                'LOWER_PHASE_HASHES': phase_3.lower_phase_hashes,
```

Comment:
Understood, and thank you for the direction on this.
- [ ] This will be rectified. :persevere: